user_manual/groupware/sync_android.rst

```diff
@@ -73,3 +73,5 @@ steps are required:
 
 
 .. tip:: DAVx⁵ lists the calendar subscriptions made through the Nextcloud Calendar app, but you need to install the `ICSx⁵ (formerly known as ICSDroid) <https://icsx5.bitfire.at/>`__ app on your Android device, `from the Google Play Store <https://play.google.com/store/apps/details?id=at.bitfire.icsdroid>`__ or `from F-Droid <https://f-droid.org/packages/at.bitfire.icsdroid/>`__ to sync them.
+
+.. tip:: With DAVx⁵ and OpenTasks installed on your mobile device, you can usually use any calendar app with CalDAV support. First, follow the steps `above <https://docs.nextcloud.com/server/19/user_manual/pim/sync_android.html#without-the-nextcloud-mobile-app>`__ to install DAVx⁵ and OpenTasks. After that, open your calendar app - such as `Etar <https://github.com/Etar-Group/Etar-Calendar>`__, if you prefer a complete FOSS solution. In the settings, look for the option to 'Add CalDAV calendar' or similar. Your previously installed DAVx⁵ will open. The authentication procedure is the same as described in `Without the Nextcloud mobile app <https://docs.nextcloud.com/server/19/user_manual/pim/sync_android.html#without-the-nextcloud-mobile-app>`__. Select **Connection with URL and username** with the ``https://sub.example.com/remote.php/dav`` URL and ``Groups are per-contact categories`` option, to connect your calendar solution with Nextcloud.
```

Comment:
static and outdated link. add and use a custom anchor https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor to have the URL generated for you